src/main/java/io/cryostat/recordings/RecordingHelper.java

```diff
@@ -945,6 +953,88 @@ private Metadata taggingToMetadata(List<Tag> tagSet) {
         return new Metadata(labels, expiry);
     }
 
+    @Blocking
+    public ActiveRecording updateRecordingMetadata(
+            Long recordingId, Map<String, String> newLabels) {
```

Comment:
This looks like it can/should be the primitive `long` instead of the boxed `Long`.